app/pages/admin/user-settings.spec.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import assert from 'assert';
+import sinon from 'sinon';
 import { shallow } from 'enzyme';
 import UserSettings from './user-settings';
 
 describe('UserSettings', function () {
   let wrapper;
-  let user = {id: 1, login: "tester"};
-  let editUser = {id: 2, login: 'volunteer'};
+  let user = {id: '1', login: "tester"};
+  let editUser = {id: '2', login: 'volunteer'};
+  sinon.stub(UserSettings.prototype, 'getUser').callsFake(() => null);
```

Comment:
Could you put this in a `before` block and add an `after` that restores the method to the original? There could be unintended side effects if the class method isn't restored in other unrelated tests